apis/apis/src/main/java/com/expensetracker/apis/controller/ExpenseEntryController.java

```diff
@@ -0,0 +1,30 @@
+package com.expensetracker.apis.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import com.expensetracker.apis.DTOs.ExpenseEntryRequest;
+import com.expensetracker.apis.services.ExpenseEntryService;
+
+@RestController
+@RequestMapping("/")
+public class ExpenseEntryController {
+
+  @Autowired
+  private ExpenseEntryService expenseEntryService;
+
+  // TODO later we add user id as path variable /users/{userid}/expenses
+  @PostMapping("apis/expenses/")
+  ResponseEntity<String> addExpenseEntry(@RequestBody ExpenseEntryRequest expenseEntryRequest) {
+    try {
+      String response = expenseEntryService.addExpenseEntry(expenseEntryRequest);
```

Comment:
Lets keep try catch in service method only, expenseEntry.addExpenseEntry will directly return ResponseEntity if created successfully, otherwise throw exception
There should be lesser code in controller